en/components/themes/component-themes.md

```diff
@@ -180,25 +179,12 @@ The below sample uses the sample from the [View Encapsulation](#view-encapsulati
 @import '~igniteui-angular/lib/core/styles/themes/utilities';
 @import '~igniteui-angular/lib/core/styles/components/avatar/avatar-theme';
 @import '~igniteui-angular/lib/core/styles/components/avatar/avatar-component'; 
-// Disable legacy support first.
-$igx-legacy-support: false;
+// Enable legacy support first.
+$igx-legacy-support: true;
 $avatar-theme: igx-avatar-theme($initials-background: royalblue);
 
 :host ::ng-deep {
-    // We still need to include the base avatar styles just once.
-    @include igx-avatar(igx-avatar-theme());
-
-    // Pass the theme to overwrite the default root vars.
-    @include igx-css-vars($avatar-theme);
-}
-```
-As a bonus, any Ignite UI for Angular theme built with the `$igx-legacy-support` set to `false` will allow styling of components without the need to use Sass in your project. For instance the above could be achieved by setting the value of `--igx-avatar-initials-background` CSS variable to the desired color:
-
-```css
-/* app-avatar.component.css */
-
-:host ::ng-deep {
-    --igx-avatar-initials-background: royalblue;
+    @include igx-avatar($avatar-theme);
```

Comment:
Please make sure this example shows how you can style view encapsulated components using CSS variable themes and without the use  of `::ng-deep`. In other words, rework the entire sample.